wremnants/qcdScaleByHelicity_helper.py

```diff
@@ -19,16 +19,8 @@
 def makeQCDScaleByHelicityHelper(is_w_like = False, filename=None):
     if filename is None:
         #
-        # FIXME: for tests with binning, to be removed eventually
-        #
-        # if is_w_like:
-        #     filename = f"{data_dir}/angularCoefficients/z_coeffs.pkl.lz4"
-        #     filename = f"{data_dir}/angularCoefficients/z_coeffs_testVpt.pkl.lz4"
-        # else:
-        #     #filename = f"{data_dir}/angularCoefficients/w_coeffs.pkl.lz4"
-        #     filename = f"{data_dir}/angularCoefficients/w_coeffs_testVpt.pkl.lz4"
-        #filename = f"{common.data_dir}/angularCoefficients/w_z_coeffs.pkl.lz4"
-        filename = f"{common.data_dir}/angularCoefficients/w_z_coeffs_testVpt.pkl.lz4" # Vpt binning based on 10% quantiles (but from old aMC@NLO)
+        #filename = f"{common.data_dir}/angularCoefficients/w_z_coeffs.pkl.lz4" # Vpt binning based on common.ptV_binning
+        filename = f"{common.data_dir}/angularCoefficients/w_z_coeffs_testVpt.pkl.lz4" # Vpt binning based on common.ptV_10quantiles_binning
```

Comment:
Can you name it something that doesn't include quantiles? And added it to the repo?